packages/components/pie-toast/test/component/pie-toast.spec.ts

```diff
@@ -17,4 +35,162 @@ test.describe('PieToast - Component tests', () => {
         // Assert
         expect(toast).toBeVisible();
     });
+
+    test.describe('Props', () => {
+        test.describe('isOpen', () => {
+            test('should not render component if isOpen is false', async ({ mount, page }) => {
```

Comment:
You may also want a test for when it is `true` e.g the toast is visible.

Reply:
is Open is true by default and as we talked in the meeting default attributes are usually tested in the first test.